options.go

```diff
@@ -376,6 +406,22 @@ func WithImageMount(source string, subpath string, target ContainerMountTarget)
 	}
 }
 
+// WithAlwaysPull will pull the image before starting the container
+func WithAlwaysPull() CustomizeRequestOption {
```

Comment:
suggestion: pass in the parameter so can set and unset, without two options. Also match the field name for consistency?